t51-projects/gh-project-board.php

```diff
@@ -1,45 +1,70 @@
 <?php
 
 
-function curl_fetch() {
+$url = 'https://api.github.com/users/jessLundie'; // Set source URL
 
-	$url = 'https://api.github.com/users/jessLundie'; // Set source URL
+$media_header = array(
+	"Accept: application/vnd.github.inertia-preview+json",
+); // Custom media header required for projects in beta
 
-	$column_id = "15445284"; // Column id, found in the column link here: https://d.pr/i/W7RQJB
+$user_agent = "jessLundie"; // Set user agent
 
-	$project_url = "https://api.github.com/projects/columns/" . $column_id . "/cards"; // Set project URL
+$token = ""; // Basic Auth Token
 
-	$token = "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"; // Basic Auth Token
+$column_id = "15445284"; // Column id, found in the column link here: https://d.pr/i/W7RQJB
 
-	$user_agent = "jessLundie"; // Set user agent
+$project_url = "https://api.github.com/projects/columns/" . $column_id . "/cards"; // Set API URL for project
 
-	// Custom media header required for projects in beta
-	$headers = array(
-		"Accept: application/vnd.github.inertia-preview+json",
-	);
+function curl_fetch( $url, $media_header, $user_agent, $token, $api_url ) {
```

Comment:
we happen to be using `curl` here but that's not _required_ or _inherent_ to what we're doing. can you think of a name that communicates only what we're doing and what matters to someone using this function? otherwise, what situation would we be in if we decided to use something other than `curl`? would we have to rewrite our existing code that calls this and make it call `some_other_fetch()`?